packages/driver/cypress/e2e/issues/23927.cy.js

```diff
@@ -0,0 +1,15 @@
+// https://github.com/cypress-io/cypress/issues/23927
+describe('issue 23927', () => {
+  it('Fails gracefully if origin page throws undefined', () => {
+    cy.visit('http://barbaz.com:3500/fixtures/generic.html')
+    cy.origin('http://foobar.com:3500', () => {
+      Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
@AtofStryker If you have a global `Cypress.on` handler, will it not catch any uncaught:exceptions from origin? Is this documented somewhere?

Reply:
I believe it is but I am not sure where, which tells me if it is, it is not obvious.

Reply:
@AtofStryker Can you update the docs to include this information here: https://docs.cypress.io/api/cypress-api/catalog-of-events#Uncaught-Exceptions I think this was the main problem people were running into since they just wanted to ignore it.

Reply:
added here https://github.com/cypress-io/cypress-documentation/pull/5821